cupy/_statistics/order.py

```diff
@@ -283,7 +290,10 @@ def _quantile_is_valid(q):
     return True
 
 
-def percentile(a, q, axis=None, out=None, *, method='linear', keepdims=False,
+def percentile(a, q, axis=None, out=None, *,
+               overwrite_input=False,
+               method='linear',
+               keepdims=False,
```

Comment:
Align with NumPy's signature.

```suggestion
def percentile(a, q, axis=None, out=None,
               overwrite_input=False,
               method='linear',
               keepdims=False,
               *,
```